README.rst

```diff
@@ -18,10 +18,15 @@ This repository provides a simple "hello world" application example for `REANA
 <http://reanahub.io/>`_ reusable research data analysis plaftorm. The example
 generates greetings for persons specified in an input file.
 
-Making a research data analysis reproducible means to provide "runnable recipes"
-addressing (1) where the input datasets are, (2) what software was used to
-analyse the data, (3) which computing environment was used to run the software,
-and (4) which workflow steps were taken to run the analysis.
+Analysis structure
+==================
+
+Making a research data analysis reproducible basically means to provide
+"runnable recipes" addressing (1) where is the input data, (2) what software was
+used to analyse the data, (3) which computing environments were used to run the
+software and (4) which computational workflow steps were taken to run the
+analysis. This will permit to instantiate the analysis on the computational
+cloud and run the analysis to obtain (5) output results.
 
 1. Input dataset
 ================
```

Comment:
Sections 1, 2 etc should be underlined only with '-' as in world population analysis.

See also the different header ("REANA example" without "demo").